pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -487,7 +487,15 @@ func (l *LibvirtDomainManager) preStartHook(vmi *v1.VirtualMachineInstance, doma
 		return domain, fmt.Errorf("preparing ephemeral container disk images failed: %v", err)
 	}
 	// Create images for volumes that are marked ephemeral.
-	err = l.ephemeralDiskCreator.CreateEphemeralImages(vmi)
+	isDevEphemeralBackingSource := make(map[string]bool)
+	for _, disk := range domain.Spec.Devices.Disks {
+		if disk.BackingStore != nil && disk.BackingStore.Source != nil {
+			if disk.BackingStore.Source.Dev != "" && disk.BackingStore.Source.Name != "" {
+				isDevEphemeralBackingSource[disk.BackingStore.Source.Name] = true
+			}
+		}
+	}
```

Comment:
Would it be better to just use the existing [`isBlockDeviceVolume()`](https://github.com/kubevirt/kubevirt/blob/1bbb544d2287ec929dc5617e4c4d0a01d18f2e5a/pkg/virt-launcher/virtwrap/manager.go#L996) function? Anyway it would be cleaner to extract the `isDevEphemeralBackingSource ` creation logic to some utility function.

Reply:
I extracted the function as requested